projects/app/src/client/ui/SplashScreen.tsx

```
@@ -0,0 +1,83 @@
+
+// 'Splashscreen.setOptions' cannot be used in Expo Go. To customize the splash
+// screen, you can use development builds. [Component Stack]
+if (!isExpoGo) {
+  // Set the animation options. This is optional.
+  ExpoSplashScreen.setOptions({
+    duration: 1000,
+    fade: true,
+  });
+}
+
+export const SplashScreen = ({}: { children?: never }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)